assembly/__tests__/env-storage.spec.ts

```diff
@@ -0,0 +1,70 @@
+// This file is aim to test the env storage functions which are external functions
+
+import { env } from '../env';
+import { resetStorage } from '../vm-mock';
+import { Address, Storage } from '../std';
+import { stringToBytes } from '@massalabs/as-types';
+
+const testAddress = new Address(
+  'AU12E6N5BFAdC2wyiBV6VJjqkWhpz1kLVp2XpbRdSnL1mKjCWT6oR',
+);
+
+beforeEach(() => {
+  resetStorage();
+});
+
+describe('Testing env storage functions', () => {
+  test('Testing del (assembly_script_delete_data)', () => {
+    const key: string = 'test';
+    const value: string = 'value';
+    // given
+    Storage.set(key, value);
+    expect(Storage.get(key)).not.toBeNull();
+    // when
+    env.del(stringToBytes(key));
+    // then
+    expect(Storage.get(key)).toBeFalsy();
```

Comment:
This will panic if executed in a smart contract. You should :
- panic if the get key doesn't exist
- use has in you code instead to match your test

Reply:
I replaced by has(key) 
And throw an error in vm side